### ISSUE TITLE
Staging Review finding: Eligibility "X" does not have alt text. 

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#139035]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio VEBT
**Product:** Application for Reimbursement of Preparatory (Prep) Course for Licensing or Certification Test
**Feature:** VA Form 22-10272

## Findings details

**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

First: The "X" does not have a text alternative, meaning that screen reader and other assistive technology users are not getting the same experience as sighted users. However, the elegibility statement paired with the "X" appears to be a double negative. The "X" appears to reinforce the statement, rather than indicate that the statement isn't true. This may confuse users, especially older users or users with cognitive disabilities. 

See: https://design.va.gov/patterns/help-users-to/check-eligibility. Tagging in @DanielleThierryUSDSVA as they had weighed in on this exact issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/135213

+ @humancompanion-usds - this pattern may need some work (guidance)

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="276" height="249" alt="screenshot eligibility double negative" src="https://github.com/user-attachments/assets/0e28562a-f73f-4cd7-b051-bec6ab1faaab" />

</details>

## Recommended action

Provide alt text for the "X", or screen reader only text, that depending on the content recommendations, aligns to providing context to the statement. 

Another issue with this pattern is that there's no programmatic relationship between the icon and the statement. Screen reader users navigating this content will experience the elements as separate - whereas sighted users can identify the relationship through visual means - proximity and location. 

To bolster the programmatic relationship between the two elements, you may consider using a definition list, which provides a direct relationship between two items, which is communicated to assistive technology. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/dl
```
<dl>
  <dt>X</dt>
  <dd>Have previously applied...</dd>
</dl>
```

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.1.1 Non-text Content - Level A](https://www.w3.org/TR/WCAG22/#non-text-content)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by shiragoodman** [comment]
@tbaker1026 can you please confirm that the DST issue Matt linked is a blocker for this ticket? If so, please add as a blocker via Relationships and add the `design-system-review` label. Thanks!

**Comment by humancompanion-usds** [comment]
This is not a blocker. The finding is narrowly about alt text. Jason's ask was for us to overhaul the pattern.
